packages/blob/src/test/index.ts

```diff
@@ -36,17 +42,18 @@ describe( 'downloadBlob', () => {
 	const createElementSpy = jest
 		.spyOn( global.document, 'createElement' )
 		.mockReturnValue( mockAnchorElement );
+
 	const mockBlob = jest.fn();
-	const blobSpy = jest.spyOn( window, 'Blob' ).mockReturnValue( mockBlob );
+	const blobSpy = jest
+		.spyOn( window, 'Blob' )
+		.mockReturnValue( mockBlob as unknown as Blob );
 	jest.spyOn( document.body, 'appendChild' );
 	jest.spyOn( document.body, 'removeChild' );
 	beforeEach( () => {
 		// Can't seem to spy on these static methods. They are `undefined`.
 		// Possibly overwritten: https://github.com/WordPress/gutenberg/blob/trunk/packages/jest-preset-default/scripts/setup-globals.js#L5
-		window.URL = {
-			createObjectURL,
-			revokeObjectURL,
-		};
+		window.URL.createObjectURL = createObjectURL;
+		window.URL.revokeObjectURL = revokeObjectURL;
```

Comment:
This is a change in behavior. Before, the object was replaced with a different object. Later, the original object was restored.

In this change, some properties of the object are changed, mutating the original object. That means that the restoration does not actually do anything.